test/functional/mempool_persist.py

```diff
@@ -112,5 +112,17 @@ def run_test(self):
         assert_raises_rpc_error(-1, "Unable to dump mempool to disk", self.nodes[1].savemempool)
         os.remove(mempooldotnew1)
 
+        NUM_TXS_IN_MEMPOOL = 900
+        self.log.debug("Send %d transactions from node0 (to its own address)" % NUM_TXS_IN_MEMPOOL)
+        self.stop_nodes()
+        self.start_node(0)
+        self.nodes[0].generate(12)
+        for i in range(NUM_TXS_IN_MEMPOOL):
+            self.nodes[0].sendtoaddress(self.nodes[0].getnewaddress(), Decimal("0.00001"))
```

Comment:
How will this not create too long mempool chains?

Reply:
I'm not sure I understand the question. What's the problem?

Reply:
Currently the mempool only accepts a chain of up to 26 unconfirmed transactions, since coin selection is not deterministic, I assume we will run into travis failures at some point.

Also, IIRC the wallet is extremely slow with coin selection, when a lot of the coins are unconfirmed change. There might be some helper functions in the test framework that can create a ton of spam transactions.